config/services.yaml

```diff
@@ -30,3 +30,84 @@ services:
         arguments: ['@Twig\Extra\Intl\IntlExtension']
         tags:
             - {name: twig.extension}
+
+    bitbag_sylius_cms_plugin.twig.extension.block:
```

Comment:
We should put some comments about why do we need to override these services. Maybe we can also put them in a separate file?